File: Folder Python/Barang.py
class Barang:
    def __init__(self, id=0, nama="", kategori="", no_part="", manufaktur="", harga=0):
        # default constructor and parameterized in one
        self.id = id
        self.nama = nama
        self.kategori = kategori
        self.no_part = no_part
        self.manufaktur = manufaktur
        self.harga = harga

    # Getter and Setter for id
    def getId(self):
        return self.id

    def setId(self, value):
        self.id = value

    # Getter and Setter for nama
    def getNama(self):
        return self.nama

    def setNama(self, value):
        self.nama = value

    # Getter and Setter for kategori
    def getKategori(self):
        return self.kategori

    def setKategori(self, value):
        self.kategori = value

    # Getter and Setter for no_part
    def getNoPart(self):
        return self.no_part

    def setNoPart(self, value):
        self.no_part = value

    # Getter and Setter for manufaktur
    def getManufaktur(self):
        return self.manufaktur

    def setManufaktur(self, value):
        self.manufaktur = value

    # Getter and Setter for harga
    def getHarga(self):
        return self.harga

    def setHarga(self, value):
        self.harga = value

File: Folder PHP/Barang.php
<?php
class Barang {
    private int $id;
    private string $nama;
    private string $kategori;
    private string $no_part;
    private string $manufaktur;
    private int $harga;
    private string $image; // new attribute

    // Constructor tanpa parameter
    public function __construct(
        int $id = 0,
        string $nama = "",
        string $kategori = "",
        string $no_part = "",
        string $manufaktur = "",
        int $harga = 0,
        string $image = ""
    ) {
        $this->id = $id;
        $this->nama = $nama;
        $this->kategori = $kategori;
        $this->no_part = $no_part;
        $this->manufaktur = $manufaktur;
        $this->harga = $harga;
        $this->image = $image;
    }

    // Getters & Setters
    public function getId(): int { return $this->id; }
    public function setId(int $v): void { $this->id = $v; }

    public function getNama(): string { return $this->nama; }
    public function setNama(string $v): void { $this->nama = $v; }

    public function getKategori(): string { return $this->kategori; }
    public function setKategori(string $v): void { $this->kategori = $v; }

    public function getNoPart(): string { return $this->no_part; }
    public function setNoPart(string $v): void { $this->no_part = $v; }

    public function getManufaktur(): string { return $this->manufaktur; }
    public function setManufaktur(string $v): void { $this->manufaktur = $v; }

    public function getHarga(): int { return $this->harga; }
    public function setHarga(int $v): void { $this->harga = $v; }

    public function getImage(): string { return $this->image; }
    public function setImage(string $v): void { $this->image = $v; }
}
